olm/README.md

```diff
@@ -0,0 +1,31 @@
+# Pre-Requisites
+
+OLM packages scripts are using some required dependencies that need to be installed
+ - [curl](https://curl.haxx.se/)
+ - [https://github.com/kislyuk/yq](https://github.com/kislyuk/yq) and not [http://mikefarah.github.io/yq/](http://mikefarah.github.io/yq/)
+ - [Operator SDK v0.8.2](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md)
```

Comment:
If we're locked down to 0.787.2, maybe we should link to https://github.com/operator-framework/operator-sdk/blob/v0.8.2/doc/user/install-operator-sdk.md instead of master = 0.9.0 ?

Reply:
fixed